### DISCUSSION TITLE
Installing X64 bits under Win11 Arm will automatically install Arm64, will it be like this in the future?

### DISCUSSION THREAD

**Discussion by sln162** [MicrosoftEdge/WebView2Feedback#4033] [Q&A]
Today I am under Win11 ARM, and I want to compare the performance gap between x64 and arm64 webview2. I have a special way to uninstall and reinstall.
But I found that starting from version 111.0.1661.44, although the downloaded version is x64, even if I copy the past x64 version from other machines, it becomes the arm64 version after the installation is complete. (I have repeatedly confirmed many times)

Then I tried to install the previous x64 version, and found that 108/109/110, and even the x64 version of the previous version of 111, 111.0.1661.41, were still x64 after installation, and there was no such situation. But starting from the latest 111.0.1661.44 version, using x64 installation will become arm64.

Want to ask, is this expected behavior? There is nothing wrong with doing this. After all, arm64 must have the best performance under the ARM architecture, but what I want to know is whether this strategy will be carried in the future. Will future versions automatically install arm64 as long as x64 is installed?

If this is the case, we don't need to distinguish whether the arm64 platform is available. To guide users to install the offline version of webview2, we only need to distinguish between x64 and x86.

![image](https://user-images.githubusercontent.com/11196986/226275072-6daa499d-2e1f-4422-9d70-4b85616f6d4d.png)

**Comment by novac42** [comment]
Thanks for reaching out, I've assigned this to a dev that can help answer your question.

**Comment by LiangTheDev** [comment]
I'll discuss this with installer team. However, the best practice for the app should be to install arm64 on arm64 machines.

**Comment by sln162** [comment]
@LiangTheDev  Due to some reasons, such as the need to process one more file during synchronization on our side, and the fewer downloads of Arm64 files, the CDN may not have a cache and the download may be slow. Of course, this is not a big problem.
